lib/api/apiUtils/bucket/bucketDeletion.js

```diff
@@ -110,7 +110,7 @@ export function deleteBucket(bucketMD, bucketName, canonicalID, log, cb) {
 
     return async.waterfall([
         function checkForObjectsStep(next) {
-            return metadata.listObject(bucketName, null, null, null, 1, log,
+            return metadata.listObject(bucketName, { maxKeys: 1 }, log,
```

Comment:
Not really in the scope of the PR, but why do a listing of one key and not a get ?